.travis.yml

```diff
@@ -14,7 +14,7 @@ before_script:
   - . vendor/ci/travis/static-analysis.sh
 
 script:
-  - phpunit $PHPUNIT_FLAGS
+  - php vendor/bin/phpunit $PHPUNIT_FLAGS
```

Comment:
Travis handles this already and uses PHPUnit with respect to PHP version.
On PHP 5.4-5.6  is PHPUnit 4.8, on PHP 7.0 is PHPUnit 5.0. Should be seen from Travis log.

Could be so kind and remove for now? If I came across some troubles, I might add it.